lib/helper/request-store.js

```diff
@@ -66,7 +66,27 @@ const putArtifact = async (buildId, token, fileName, file) => {
     }
 };
 
+const deleteZip = async (buildId, token) => {
+    const options = {
+        url: `${baseUrl}/builds/${buildId}/ARTIFACTS/${ZIP_FILE}`,
+        method: 'DELETE',
+        headers: {
+            Authorization: token
+        },
+        retry: {
+            limit: RETRY_LIMIT
+        }
+    };
+
+    try {
+        return request(options);
+    } catch (err) {
+        throw new Error(err.response.body.toString());
```

Comment:
Maybe toString() method is not needed, because response type is not `buffer`.
https://github.com/sindresorhus/got/blob/2ac07e1b60f59e2219bd6c2809a9e40f56b146b6/source/core/response.ts#L141

Reply:
Thx.
I fixed it.